2021q3/clusteradm.adoc

```diff
@@ -14,7 +14,7 @@ The FreeBSD Cluster Administration Team consists of the people responsible for a
 - Finished installing a new mirror site in Japan, sponsored by Broadband Tower, Inc.
     - Full mirror site (ftp, pkg, git, doc, dns,...)
     - The network and machine resources for this mirror are generously sponsored by the Cloud and SDN Laboratory at BroadBand Tower, Inc., one of the Internet data center service providers in Tokyo, Japan, with 300+ Gbps international IP transit bandwidth
-- Improved the retension script used for artifact server of CI cluster to offerings a mix of the latest artifacts but also a selection of older artifacts for comparison, space permitting
+- Improved the retention script used for artifact server of CI cluster to offerings a mix of the latest artifacts but also a selection of older artifacts for comparison, space permitting
```

Comment:
Missed the first time, sorry: s/for artifact/for the artifact/ and s/offerings/offer/